### PR TITLE
Text Scaling Fixes

### PR DESCRIPTION
**Pull request by seemantk**
Below the headline this is a slight refinement on the method.  Here, there's a separation of zooming methods between pure svg elements, and javascript-driven elements (axes: mensuration signs, barlines, measure numbers).

**Optical Zoom**
(this is the lens class)

OK, since responsiveness depends on % units (and relative units) of measurements (for widths etc), that's how the pure SVG elements (voice notes and ribbons) are responsive within their containers.  The viewBox zoom is unaffected by the viewport's dimensions.

The actual dimensions of the SVGs within the container are vary according to the dataset.

**Electrical Zoom**
(this is the markings class)

So in order for the axes  to remain calibrated with the Optical Zoom, it needs to use the same percentages. Except...transform/translate doesn't know from percentages, only numbers.

So, the javascript-driven elements require a javascript-driven responsiveness.  Here, we read our svg's dimensions, and then calculate the margins, etc from percentages off those dimensions. 

The dimensions of the parent SVG correspond to the dimensions of its containing element.  Thus, we always scale, ultimately, to that viewport's dimensions -- including the text.

tl;dr text scaling is now uniform, and the code (html and js) is cleaner.

### CODE REVIEW

**Comment by seemantk** [comment]
@curran, this is ready for your review
(Note, combine-separate + reflines is still not working, as this was only to fix the current responsive behavior).

**Comment by curran** [comment]
@seemantk This is fantastic! The font size is indeed shockingly small ;)
